### DISCUSSION TITLE
Whitelist file?

### DISCUSSION THREAD

**Discussion by breeegz** [itzg/docker-minecraft-server#1794] [Q&A] [answered]
I'm trying to set up an environment where I can deploy multiple servers, but I would really like for the configuration to deploy a new one to be rather minimal.  I see on the documentation for Bedrock, there's a "whitelist" variable, but I didn't see the same options on the Java side.   Could I set the minecraft data to be relative, but statically define a single whitelist for all of my potential servers?

For example
```
- ./minecraft-data/:/data
- WHITELIST=/path/to/some/file/whitelist
```

How much does the Bedrock documentation crossover for Java Servers?   The Bedrock documentation seems a little better laid out for newbies like me.  I can experiment, but if there's somethings that aren't going to work, then I best stay away.

**Comment by itzg** [comment, marked as answer]
Do you mean this: https://github.com/itzg/docker-minecraft-server#whitelist-players ?

Admittedly the java edition docs are overwhelmingly long, but that's because it needs to handle a lot more use cases than bedrock.

**Reply by breeegz** [reply]
Thanks..  I feel like a goof, I'm sure that I skimmed the whole thing.

Maybe the different types of servers could be moved to a standalone page for clarity sake.  I could see benefit for people like me, who have no idea what any of those servers are, and just want to deploy a server in Docker or Compose.  Talk all the way through the vanilla setup, then links to other server options?  Having them nested in the middle is where I think I got tripped up.  

But maybe I'm just one guy who obviously didn't read the entire thing, so my opinion is forfeit.  I apologize for taking up your time and I thank you for your help.

**Reply by itzg** [reply]
That's a great suggestion. In case it helps, GitHub READMEs to have this navigation widget

![image](https://user-images.githubusercontent.com/988985/197409742-71112721-d007-426c-9caa-bb47383cdba5.png)
